frontend/package.json

```diff
@@ -10,6 +10,7 @@
     "test": "jest"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.3",
```

Comment:
why is emotion/react added - whats the functionality? i dont see it imported in the pages

Reply:
emotion/react currently is required everytime I try npm run dev.
I've just added this dependency because it seems like we need it? - not really to sure tho.